www/content/actions.md

```diff
@@ -79,3 +81,4 @@ command and a private key without a passphrase.
 [goreleaser-action]: https://github.com/goreleaser/goreleaser-action
 [actions]: https://github.com/features/actions
 [syntax]: https://help.github.com/en/articles/workflow-syntax-for-github-actions#About-yaml-syntax-for-workflows
+[only-new-tag]: https://github.com/goreleaser/goreleaser-action/issues/61#issuecomment-569286582
```

Comment:
I think that actually adding the example (see below) with a short description to the README would be better than linking the issue. That's not a strong opinion, though. 🙂 

```yaml
on:
  push:
    tags:
      - '*'
```

Reply:
I agree.. 🤔

Reply:
Done :)